src/ledger.ts

```diff
@@ -401,6 +401,7 @@ function ledger(this: any, options: LedgerOptions) {
     baseval?: number
     basecur?: string
     baserate?: number
+    xreward?: string
```

Comment:
you don't need to do this
just follow the example in the test:
https://github.com/senecajs/seneca-ledger/blob/f329ffedd9ab34d860e87237ec2b826306c278a3/test/basic.messages.ts#L200

in any case, you should not add project specific things to a generic utility

Reply:
Understood, thank you. I gonna use the entry.